### PR TITLE
Feature/shopping equipment

### PR DESCRIPTION
**Pull request by patrykbuzowiczsabre**

### CODE REVIEW

File: src/actions/flights/flights.ts [hunk 1]
```diff
@@ -12,7 +14,16 @@ export const findFlights = (flightData: SearchFlightData) => {
     error: bfmError
   };
 
-  return atkFetchCreator(buildBargainFinderMaxRequest(flightData), actions);
+  return async (dispatch, getState: GetState) => {
+    try {
+      loadEquipment(dispatch, getState);
```
Comment (wojciech-kumon) [inline src/actions/flights/flights.ts]: Didn't you forget to use `await` here?

File: src/actions/flights/flights.ts [hunk 2]
```diff
@@ -12,7 +14,16 @@ export const findFlights = (flightData: SearchFlightData) => {
...
+    } catch (e) {
+      dispatch(actions.start());
```
Comment (wojciech-kumon) [inline src/actions/flights/flights.ts]: Setting whole BFM error when equipment failed is not a proper way of handling this exception. In next pull request you will see that start action is not clearing itineraries.
Set new flag for equipment fail or use raw data without lookup as fallback (`equipmentType` in `ExpandedFlight.tsx`).

File: src/actions/flights/flights.ts (hunk 1 above)
Comment (patrykbuzowiczsabre) [reply src/actions/flights/flights.ts]: I did, fixed

File: src/actions/flights/flights.ts (hunk 2 above)
Comment (patrykbuzowiczsabre) [reply src/actions/flights/flights.ts]: ok, for consistency I also adjusted the similar logic in view TP - the other PR

**Pull request merged**